components/builder-web/app/package/package-sidebar/package-sidebar.component.ts

```diff
@@ -43,7 +43,7 @@ export class PackageSidebarComponent {
 
   get buildButtonLabel() {
     return this.building ? 'Build pending' :
-           this.isNewProject ? 'Build latest versions' : 'Build latest version';
+      this.isNewProject ? 'Build latest versions' : `Build latest ${this.platform.name} version`;
```

Comment:
do we need this? It does not appear to be replacing any icon but is just additional text.